android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -847,6 +859,10 @@ public WebResourceResponse shouldInterceptRequest(WebView view, WebResourceReque
           message = "Unknown";
         }
 
+        if (statusCode == 401) {
+          return null;
+        }
+
```

Comment:
@nayuki-at-work Is this ok to return null if error = 401 (authentication fail)?
If do not return null in the case, `onReceivedHttpAuthRequest` callback is not called